packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -896,14 +898,38 @@ const Conversation = WebexPlugin.extend({
         async: false
       });
     }
+    const cloneActivity = cloneDeepU(params);
```

Comment:
Why not use the `cloneDeep` from lodash that is already imported?

Reply:
lodash cloneDeep not able to copy scrObject function toJWE . It has some issue with custom object copying.

Reply:
Though I am giving chance to lodash cloneDeepWith if that works I can remove it.

Reply:
I added customizer function using cloneDeepWith that copy instance function, now not require additional library clone-deep.